sdk/tables/aztables/zc_tableConstants.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package aztables
+
+const (
+	Timestamp     = "Timestamp"
+	PartitionKey  = "PartitionKey"
+	RowKey        = "RowKey"
+	EtagOdata     = "odata.etag"
+	ETag          = "ETag"
+	OdataMetadata = "odata.metadata"
+	OdataType     = "@odata.type"
+	EdmBinary     = "Edm.Binary"
+	EdmBoolean    = "Emd.Boolean"
+	EdmDateTime   = "Edm.DateTime"
+	EdmDouble     = "Edm.Double"
+	EdmGuid       = "Edm.Guid"
+	EdmInt32      = "Edm.Int32"
+	EdmInt64      = "Edm.Int64"
+	Edm           = "Edm.String"
+	ISO8601       = "2006-01-02T15:04:05.9999999Z"
```

Comment:
Do all of these consts need to be exported? Seems like it'll pollute the exported space bit for intellisense.

Reply:
good callout - none of these should be exported.